dChatServer/ChatPacketHandler.cpp

```diff
@@ -346,45 +348,31 @@ void ChatPacketHandler::HandleChatMessage(Packet* packet) {
 	eChatChannel channel;
 	uint32_t size;
 	
-	inStream.IgnoreBytes(4);
+	inStream.IgnoreBytes(4); // padding/garbage
 	inStream.Read(channel);
 	inStream.Read(size);
-	inStream.IgnoreBytes(77);
+	inStream.IgnoreBytes(77); // padding/garbage
```

Comment:
I would consider "the structure the client uses to send this packet is shared in many chat messages that are sent to the server.  Because of this, there is a large gap of unused data here" or something along those lines just because this is an abnormally large chunk of data to just call "padding"